client/src/modules/reports/generate/stock_exit/stock_exit.config.js

```diff
@@ -0,0 +1,83 @@
+angular.module('bhima.controllers')
+  .controller('stock_exitController', InventoryFileConfigController);
+
+InventoryFileConfigController.$inject = [
+  '$sce', 'NotifyService', 'BaseReportService', 'AppCache', 'reportData', '$state',
+  'LanguageService',
+];
+
+function InventoryFileConfigController($sce, Notify, SavedReports, AppCache, reportData, $state, Languages) {
+  const vm = this;
+  const cache = new AppCache('configure_stock_exit_report');
+  const reportUrl = 'reports/stock/exit';
+
+  // default values
+  vm.includePatientExit = 1;
+  vm.previewGenerated = false;
+
+  // check cached configuration
+  checkCachedConfiguration();
+
+  vm.onSelectDepot = depot => {
+    vm.depot = depot;
+  };
+
+  vm.clear = key => {
+    delete vm[key];
+  };
+
+  vm.clearPreview = () => {
+    vm.previewGenerated = false;
+    vm.previewResult = null;
+  };
+
+  vm.preview = form => {
+    if (form.$invalid) { return 0; }
+
+    const params = {
+      depotUuid : vm.depot.uuid,
+      dateFrom : vm.dateFrom,
```

Comment:
I propose that you set `vm.dateFrom = new Date();` as a default value early in the controller to prevent the following:

![2018-04-26_23-18-39](https://user-images.githubusercontent.com/896472/39334871-7c3c5fca-49a8-11e8-8501-288db765a275.gif)
_Submission w/o date or checkboxes is interesting_